cuqi/utilities/_utilities.py

```diff
@@ -229,3 +231,88 @@ def approx_gradient(func, x, epsilon= 0.000001):
         eps_vec[i] = 0.0
         
     return FD_gradient
+
+
+# Function for plotting 1D density functions
+def plot_1D_density(density:Density,
+                    v_min, v_max,
+                    N=500):
```

Comment:
```suggestion
                    N=501):
```

Reply:
Suggesting 501 to have have 500 steps by default, aligned with the 2D case of using 201 points.